recipe/meta.yaml

```diff
@@ -36,7 +36,7 @@ requirements:
     - cgal-cpp >=5.6
     - gmp   # [unix]
     - mpfr
-    - mpir  # [win]
+    - gmp  # [win]
```

Comment:
We should merge the 2 lines with gmp.
As an alternative, we could omit gmp completely and rely on the fact that it is a dependency of mpfr, like cgal-cpp does. See also #55.